### DISCUSSION TITLE
Is it possible to pass IP address to the server?

### DISCUSSION THREAD

**Discussion by christian-saldana** [alvr-org/ALVR#1108] [General]
From other discussions I learned there no longer is a UI or a command line option to specify the server IP. However, if the repo was forked could you still pass the IP address to the client and connect to the server? Does the previous code still exist to make this possible?

**Comment by zmerp** [comment]
You can't specify the server IP in the client anymore, but you can specify the client IP in the server. Use the button "Add client manually" in the Connection tab.

**Reply by christian-saldana** [reply]
Yeah I understand that, but I am wanting to pass it through the client. I just want to know how difficult it would be to pass the server IP address to the client to connect. Ideally, the way it would work is that there is a UI the user can use or there is an API request to a server that would then return the IP Address to connect to.  The client used to pass the server IP and connect, can it still do that with an adjustment to the code?

**Reply by zmerp** [reply]
The codebase changed drastically. The only code kept from the old repository is for the encoder, decoder and white lobby room. The rest of the code, all network code has been rewritten. There is plan to add a user interface on the client for some features like the one you are asking but it's very low on priority at the moment.

**Reply by christian-saldana** [reply]
I see. Thank you for letting me know.
